etc/ci/performance/test_all_gecko.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
Let's combine this script with `test_all.sh` and have it take a parameter for the engine to use.